.travis.yml

```diff
@@ -32,19 +32,19 @@ matrix:
   include:
     - jdk: oraclejdk8
       env:
-        - FLINK_VERSION="1.3.0" SCALA_VERSION="2.11"
+        - FLINK_VERSION="1.5.1" SCALA_VERSION="2.11"
         - CACHE_NAME=JDK8_F130_A
     - jdk: oraclejdk8
       env:
-        - FLINK_VERSION="1.3.0" SCALA_VERSION="2.10"
+        - FLINK_VERSION="1.5.1" SCALA_VERSION="2.10"
```

Comment:
the version on scala 2.10 was deprecated on flink version 1.4.0
you have to remove this test